_clients/java-rest-high-level.md

```diff
@@ -25,7 +22,33 @@ To start using the OpenSearch Java high-level REST client, ensure that you have
 
 You can now start your OpenSearch cluster. The OpenSearch 1.x high-level REST client works with the 1.x versions of OpenSearch.
 
-The following example uses credentials that come with the default OpenSearch configuration. If you’re using the high-level REST client with your own OpenSearch cluster, be sure to change the code to use your own credentials.
+## Security
```

Comment:
Reading the content, doesn't seem like it needs a separate section`##Security`. The content here should also part of the `##Setup`.

Reply:
I would argue that we should keep `Setup` and `Security` separate. Our intention for `Setup` is only to explain users can add the client to their IDE/whatever tool so they can start using the client to connect to a cluster, whereas `Security` is strictly for explaining how to securely connect to a cluster. If users have their own security configurations, they can ignore the `Security` section, but including it inside `Setup` doesn't directly call out the required security code, which we're trying to do.

Reply:
OpenSearch comes with security installed so users cannot ignore this section unless they have explicitly disabled the security plugin or using the minimal distribution. 

If you want to keep this separate, call out these are configurations are related to the security plugin.